lib/ansible/modules/cloud/misc/helm_cli.py

```diff
@@ -0,0 +1,542 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm_cli
+short_description: Manages Kubernetes packages with the Helm package manager
+description:
+  - Install, upgrade, delete packages with the Helm package manager.
+version_added: "2.10"
+author:
+  - Lucas Boisserie (@LucasBoisserie)
+notes:
+  - Support for helm3 is not finished waiting fix on helm 3
+requirements:
+  - "helm"
+  - "yaml"
+  - "tempfile"
```

Comment:
Isn't `tempfile` part of the Python standard library? Or do you refer to something else?

Reply:
Yes, exactly !
I removed it